### ISSUE TITLE
MOT with bbox outputs from MMdetection

### ISSUE THREAD

**Issue by platojinsol** [open-mmlab/mmtracking#334]
Hi, I was wondering if it's possible to use detection outputs from pretrained models provided by MMdetection for MOT. I found a relevant question from the past (#127 replied by @OceanPang) but there wasn't much details regarding the annotation files to provide. MMdetection outputs detection bounding boxes for each image, but should the annotation.pkl file to be included in config have bounding boxes from all frames? Any guidance would be appreciated.

**Comment by GT9505** [comment]
Yes. The MOTChallange dataset support loading detections from [mmdet](https://github.com/open-mmlab/mmtracking/blob/master/mmtrack/datasets/mot_challenge_dataset.py#L42).

You can use models from mmdetection to get all bboxes of all images. Typically, the bboxes will be saved as a pkl file.
Then you can pass the pkl file of detections to [`detection_file`](https://github.com/open-mmlab/mmtracking/blob/master/configs/mot/deepsort/deepsort_faster-rcnn_fpn_4e_mot17-public-half.py#L26).
